src/pages/LandingPage.tsx

```diff
@@ -4,7 +4,11 @@ import {
   Form,
   Page,
   PageSection,
-  PageSectionVariants, Pagination, PaginationProps,
+  PageSectionVariants, 
+  Pagination,
+  PageSectionVariants, 
+  Pagination, 
+  PaginationProps,
```

Comment:
I tried fixing the confilict but something happened here :grin: 

```suggestion
  PageSectionVariants, 
  Pagination, 
  PaginationProps,
```